### ISSUE TITLE
Add project's instance URL in repository description

### ISSUE THREAD

**Issue by finn0** [sitaramc/git-notes#4]
@sitaramc, Could you please add the project's instance [URL](https://gitolite.com/gcs.html) in repository description? This would be helpful for people who discover this project; they don't have to search URL to view the content.

**Comment by sitaramc** [comment]
On Tue, Nov 19, 2019 at 04:51:42AM -0800, Vipul wrote:
> @sitaramc, Could you please add the project's instance [URL](https://gitolite.com/gcs.html) in repository description? This would be helpful for people who discover this project; they don't have to search URL to view the content.

gcs is not the only page that is accessible from git-notes, as
far as I remember (I have to tell you it's been ages since I
looked at these pages).

Maybe I misunderstood; could you be more precise?  Sorry I have
had a long day at work and my brain refuses to work any more :-(

**Comment by finn0** [comment]
@sitaramc, Where do you host source code of [GCS](https://gitolite.com/gcs.html)? I want to do some minor fixes in GCS.

**Comment by sitaramc** [comment]
On Tue, Nov 19, 2019 at 05:37:42AM -0800, Vipul wrote:
> @sitaramc, Where do you host source code of [GCS](https://gitolite.com/gcs.html)? I want to do some minor fixes in GCS.

this very repo; subdirectory "what"

**Comment by sitaramc** [comment]
On Tue, Nov 19, 2019 at 05:37:42AM -0800, Vipul wrote:
> @sitaramc, Where do you host source code of [GCS](https://gitolite.com/gcs.html)? I want to do some minor fixes in GCS.

I see the confusion.  It's partly because I've redirected
"sitaramc.github.com" to "gitolite.com", and since gitolite is
the main project I write/support, and the rest is just help
pages for git in general, that's the main "index".

The index page for the rendered version of git-notes is
"gitolite.com/index2.html".

I need to fix this.  I think I created a github organisation
called gitolite; I should use that exclusively for gitolite, and
then the git-notes page can be properly on
"sitaramc.github.com/git-notes" or whatever the gh-pages URL is.

I set all this up so long ago I've forgotten all the moving
parts, and what goes where.  Will take some time, especially
because $DAYJOB is hell right now.
